### DISCUSSION TITLE
Unable to migrate account from mastodon.social to fosstodon.org

### DISCUSSION THREAD

**Discussion by CorruptComputer** [mastodon/mastodon#19607] [General]
I am trying to migrate from mastodon.social to fosstodon.org, however when I try the migration process it seems to be broken.

On fosstodon.org I setup the account alias:
![image](https://user-images.githubusercontent.com/5573038/199102342-354f6d2d-6024-4abb-9f29-ef8398706477.png)

On mastodon.social I try to migrate and it does not work:
![image](https://user-images.githubusercontent.com/5573038/199102425-d77d658d-79a3-4f3a-8d18-da9a2527edf8.png)

**Comment by CorruptComputer** [comment]
I think this might be caused by them being on different mastodon versions, mastodon.social is on v4.0.0rc1 and fosstodon.org is on v3.5.3. Is backwards compatibility for these features supposed to work? I would imagine that not all servers are going to update at the same time, so this feels fairly important to have to me.

**Reply by Gargron** [reply]
I can assure you compatibility is not a problem but mastodon.social is straight up having technical issues since yesterday, the Sidekiq queue has 1.4 million items in it, so anything that requires asynchronous processing will be massively delayed.

**Reply by CorruptComputer** [reply]
Yeah I know there was an outage there yesterday, which is the reason I want to switch off it. Shouldn't the account migration be checking with the remote server when you submit the request? That seems like a bit of an oversight to have it only process those asynchronously since it can lead to situations like this where my followers are just stuck there and I can't migrate them.
